news/2022-08-25-changes-to-osu-taiko-sr-and-pp.md

```diff
@@ -0,0 +1,149 @@
+
+We will also be using terminology coined by the osu!taiko community to refer to how patterns are placed. **k** refers to kat (blue), **d** refers to don (red), coined as different 'colours'. Long streams that are of one colour are referred to as *monos*, while notes that are surrounded by brackets () refer to **1/6** spacing, and notes inside square brackets [] refer to **1/8** spacing. 
+
+If you would like to know more about the changes posted, or for more concise information on the some of the in development changes, head on over [the global osu!taiko rework document](https://docs.google.com/document/d/1Z5GC4DMqOVzeIERMSK3qpQaqjq-sVnhbuoxAwy9qxDs/edit#). This is a working document, and will periodically be updated with the latest, (and greatest) developments.
+
+## An insight into the base of the new rework
+
+Previous rework efforts have aimed to tackle all three skills at once, which has made it difficult to collect meaningful and targetted feedback. As such, the osu!taiko pp committee opted for a modular approach, tackling new changes and reported issues one stage at a time, in order to develop more accurate solutions. In this osu!taiko star rating rework, we have re-evaluated the stamina and colour components of difficulty, with the capacity for changes following community feedback. In addition, the performance points system has been adjusted to bring it closer in line with community expectations. During this time, we have been and will continue to work on other difficulty components (both small- and large-scale) in preparation for future releases.
+Moving forward, osu!taiko star rating will be receiving numerous amounts of changes, potentially placing star rating and performance points in a state of flux, particularly over the coming months. This is generally desirable as we would want difficulty to accurately reflect the latest trends, but we understand and apologise if this may bring some unintended complications or frustrations, especially in the short term. 
+
+The previous rework effort aimed to the tackle all three skills at once. This not only made it difficulty to collect meaningful and targetted feedback, but halted continuous development efforts. To combat this, the osu!taiko pp committee developers ([vun](https://osu.ppy.sh/users/6932501) and [ltca](https://osu.ppy.sh/users/11475208)) opted for a more modular approach, tackling the new changes one stage at a time, in hopes for being able to fix reported issues one stage at a time. 
```

Comment:
From what I learned since starting to contribute, you are only supposed to link to players the first time they are mentioned, apologies if this is not correct!

```suggestion
The previous rework effort aimed to the tackle all three skills at once. This not only made it difficulty to collect meaningful and targetted feedback, but halted continuous development efforts. To combat this, the osu!taiko pp committee developers (vun and ltca) opted for a more modular approach, tackling the new changes one stage at a time, in hopes for being able to fix reported issues one stage at a time. 
```

Reply:
will know for the future! also done in 1fa2745